includes/wp-admin/class-settings-fields.php

```diff
@@ -478,7 +486,37 @@ public static function render_outbox_purge_days_field() {
 	}
 
 	/**
-	 * Render use hashtags field.
+	 * Render allow interactions field.
+	 */
+	public static function render_allow_interactions_field() {
+		if ( defined( 'ACTIVITYPUB_DISABLE_INCOMING_INTERACTIONS' ) && ACTIVITYPUB_DISABLE_INCOMING_INTERACTIONS ) {
+			echo '<p class="description">' . \esc_html__( '⚠ This setting is defined through server configuration by your blog&#8217;s administrator.', 'activitypub' ) . '</p>';
+			return;
+		}
+
+		$allow_likes   = get_option( 'activitypub_allow_likes', '1' );
+		$allow_reposts = get_option( 'activitypub_allow_reposts', '1' );
+		?>
+		<fieldset>
+			<p>
+				<label>
+					<input type="checkbox" name="activitypub_allow_likes" value="1" <?php checked( '1', $allow_likes ); ?> />
+					<?php esc_html_e( 'Receive likes', 'activitypub' ); ?>
+				</label>
+			</p>
+			<p>
+				<label>
+					<input type="checkbox" name="activitypub_allow_announces" value="1" <?php checked( '1', $allow_reposts ); ?> />
+					<?php esc_html_e( 'Receive reblogs', 'activitypub' ); ?>
```

Comment:
```suggestion
					<?php esc_html_e( 'Receive reblogs (also known as boosts)', 'activitypub' ); ?>
```

I like that we use a more general term, but maybe we should mention that Mastodon calls them "boosts".

Reply:
```suggestion
					<?php esc_html_e( 'Receive reblogs (boosts)', 'activitypub' ); ?>
```
Would this provide enough context?